src/backends/dict.jl

```diff
@@ -85,14 +85,14 @@ Base.isempty(Δs::DictFIs) = isempty(Δs.dict)
 Base.length(Δs::DictFIs) = length(Δs.dict)
 Base.iszero(Δs::DictFIs) = isempty(Δs) || all(iszero.(Δs.dict))
 function StochasticAD.derivative_contribution(Δs::DictFIs{V}) where {V}
-    sum((Δ * event.w for (event, Δ) in pairs(Δs.dict)), init = zero(V))
+    sum((Δ * event.w for (event, Δ) in pairs(Δs.dict)), init = zero(V) * 0.0)
```

Comment:
What was the issue here with only `zero(V)`? Isn't `*0.0` bad if, e.g., a different floating point precision is used?

Reply:
The type of `Δ` is V`, but I multiply by 0.0 since the type of this sum should be `typeof(Δ * event.w)` and `w` is currently hardcoded to `Float64`.